torch/_dynamo/config.py

```diff
@@ -130,21 +130,8 @@
 # Disable dynamo
 disable = os.environ.get("TORCH_COMPILE_DISABLE", False)
 
-# If a PyTorch module is in this allowlist, torchdynamo will be allowed
-# to inline objects from it or its children.
-skipfiles_inline_module_allowlist = {
-    torch.nn,
-    torch.distributions,
-    torch.testing,
-    torch.ao.nn,
-    torch._refs,
-    torch._prims,
-    torch._decomp,
-    torch.utils._contextlib,
-    torch.utils._pytree,
-    torch.fx._pytree,
-    torch.sparse,
-}
+# legacy config, does nothing now!
+skipfiles_inline_module_allowlist = {}
```

Comment:
I think the skip/inline rules should be defined by developer rather than users, so I moved this to ```skipfiles.py```. If we really need to support users side skip/inline rules, we can extend the current list by adding custom skip/inline lists.

Reply:
confirmed no use sites via fbgs

Reply:
Yes